org.flightgear.FlightGear.yaml

```diff
@@ -90,7 +90,7 @@ modules:
 - name: plib
   sources:
   - type: archive
-    url: https://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+    url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
```

Comment:
Any reason for such change ? It's surely better to use https rather than unencrypted http.

Reply:
https unavailable now...

Reply:
Since there's the checksum it isn't really a big problem.